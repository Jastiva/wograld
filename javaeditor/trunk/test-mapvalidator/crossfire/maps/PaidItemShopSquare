arch map
name test for ShopSquare map validator
difficulty 1
width 6
height 4
msg
Created:  2010-06-27 Andreas Kirschbaum
Modified: 2010-11-05 ank
endmsg
end
arch shop2
end
arch rod1
end
arch shop2
y 1
end
arch shop2
y 2
end
arch shop2
y 3
end
arch mob1
y 3
end
arch shop2
x 1
end
arch shop2
x 1
y 1
end
arch rod1
x 1
y 1
no_pick 0
end
arch shop2
x 1
y 2
end
arch rod1
x 1
y 2
no_pick 1
end
arch shop2
x 1
y 3
end
arch shop2
x 2
end
arch shop2
x 2
y 1
end
arch rod1
x 2
y 1
unpaid 0
end
arch shop2
x 2
y 2
end
arch rod1
x 2
y 2
unpaid 1
end
arch shop2
x 2
y 3
end
arch shop2
x 3
end
arch shop2
x 3
y 1
end
arch rod1
x 3
y 1
monster 0
end
arch shop2
x 3
y 2
end
arch rod1
x 3
y 2
monster 1
end
arch shop2
x 3
y 3
end
arch shop2
x 4
end
arch shop2
x 4
y 1
end
arch rod1
x 4
y 1
generator 0
end
arch shop2
x 4
y 2
end
arch rod1
x 4
y 2
generator 1
end
arch shop2
x 4
y 3
end
arch shop2
x 5
end
arch shop2
x 5
y 1
end
arch rod1
x 5
y 1
weight 0
end
arch shop2
x 5
y 2
end
arch rod1
x 5
y 2
weight 1
end
arch shop2
x 5
y 3
end
