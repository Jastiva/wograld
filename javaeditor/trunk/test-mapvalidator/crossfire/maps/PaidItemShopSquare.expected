[0|0] Shop square contains paid item
[1|1] Shop square contains paid item
[2|1] Shop square contains paid item
[3|1] Shop square contains paid item
[4|1] Shop square contains paid item
[5|2] Shop square contains paid item
